public/contact us .html

```diff
@@ -83,34 +84,45 @@ <h4>AEC Library</h4>
 	   </div>
 	   <div class="contact-form">
 		   <h2><strong> Send a Message</strong></h2>
+		   <form onsubmit="return false">
 		   <div class="form">
 			   <div class="inputBox w50">
-				   <input type="text"  name="" required>
+				   <input type="text"  name="firstname" required>
 				   <span>First Name</span>
+				   <span class = "error"> </span>
 			   </div>
 			   <div class="inputBox w50">
-				   <input type="text" name="" required>
-				   <span>Last Name</span>
+				   <input type="text" name="lastname" required>
+				   <span >Last Name </span>
+				   <span class="error"> </span>
 			   </div>
 			   <div class="inputBox w50">
-				   <input type="text" name="" required>
+				   <input type="email" name="email" required>
 				   <span>Email Address</span>
+				   <span class = "error"></span>
 			   </div>
 			   <div class="inputBox w50">
-				   <input type="text"  name="" required>
+				   <input type="number"  name="mobilenumber" required>
 				   <span>Mobile Number</span>
```

Comment:
Use `type="tel"`, as they are the type for mobile phones, `number` doesn't make sense...
Secondly handle it for 10 digits...